### ISSUE TITLE
toolchain fails to recognize option --sdk

### ISSUE THREAD

**Issue by Dinsmoor** [kivy/buildozer#524]
Attemping to run $ buildozer -v android debug
results in the below:

`
BUILD FAILED	
/home/d/.buildozer/android/platform/android-sdk-20/tools/ant/build.xml:1037: The following error occurred while executing this line:	
/home/d/.buildozer/android/platform/android-sdk-20/tools/ant/build.xml:1039: The following error occurred while executing this line:	
/home/d/.buildozer/android/platform/android-sdk-20/tools/ant/build.xml:336: Value for 'output' is not valid. It must resolve to a single path	
	
Total time: 2 seconds
[INFO]:    STDERR:
	
[INFO]:    COMMAND:
cd /home/d/Documents/python/tg/.buildozer/android/platform/build/dists/tmid && /home/d/.buildozer/android/platform/apache-ant-1.9.4/bin/ant debug

[WARNING]: ERROR: /home/d/.buildozer/android/platform/apache-ant-1.9.4/bin/ant failed!
Command failed: /usr/bin/python2.7 -m pythonforandroid.toolchain apk --debug --bootstrap=sdl2 --dist_name tmid --name 'TMID' --version 0.1 --package info.pappad.tmid --android_api 19 --sdk 19 --minsdk 9 --private /home/d/Documents/python/tg/.buildozer/android/app --permission INTERNET --permission WRITE_EXTERNAL_STORAGE --orientation portrait --window --copy-libs --arch armeabi-v7a --color=always --storage-dir=/home/d/Documents/python/tg/.buildozer/android/platform/build

`

Running the failed command results in error:
`
toolchain.py: error: ambiguous option: --sdk could match --sdk-dir, --sdk_dir
`

**Comment by Dinsmoor** [comment]
A note for the above, it's all from a fresh install of kivy, buildozer as listed on the Ubuntu installation section of the kivy website.
Ubuntu 16.04

**Comment by inclement** [comment]
Thanks for the report. I've seen it, but I'm not sure offhand what is wrong. I'll try to look into it.

**Comment by Dinsmoor** [comment]
It was a python-for-android versioning issue I got from pip. After installing the newest one from the python-for-android master branch, I was able to get past that step.

Going to close this issue, since this ticket is specific to that problem, but FYI the entire buildozer build process is full of strange "Build failed", but if you run them manually then sometimes they work, sometimes they don't. Maybe it's time to go from start-to-finish for an example kivy app built for android using nothing but the documentation of the Kivy website and these projects, and see if any problems that may be easy fixes come up. It might be refreshing.

Cheers.
